src/config/util.js

```diff
@@ -10,4 +10,11 @@ export const requiredEnv = (name) => {
 /** @type {"development" | "production" | "test"} */
 // @ts-ignore
 export const nodeEnv = process.env.NODE_ENV ?? 'development';
-export const serverId = randomUUID();
+
+/** @type {string} */
+let serverId;
+export const getServerId = () => {
+  if (!serverId) serverId = randomUUID();
+
+  return serverId;
+};
```

Comment:
Why you don't just return UUID?

Reply:
Because it is imported in multiple files and without this function, every import received a different id.
It behaves as if every import reruns the whole file, calling `randomUUID` function again.
Spotted it in `test/account/transfer.test.js` - bus and server received different `serverId` and as a result, communication was broken.